common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -248,19 +256,9 @@ private boolean processWebsiteRequest(@NonNull final WebView view, @NonNull fina
             //       Until that comes, we'll only handle this in ipphone.
             if (packageHelper.isPackageInstalledAndEnabled(applicationContext, IPPHONE_APP_PACKAGE_NAME) &&
                     IPPHONE_APP_SIGNATURE.equals(packageHelper.getCurrentSignatureForPackage(IPPHONE_APP_PACKAGE_NAME)) &&
-                    packageHelper.isPackageInstalledAndEnabled(applicationContext, COMPANY_PORTAL_APP_PACKAGE_NAME)) {
+                    packageHelper.isPackageInstalledAndEnabled(applicationContext, COMPANY_PORTAL_PROD_APP_PACKAGE_NAME)) {
```

Comment:
I'm assuming that this means that we don't want to link into BrokerHostApp.

Reply:
That's correct.

Reply:
Can we say that here?